schema/card_schema.json

```diff
@@ -118,7 +118,7 @@
             ]
         },
         "pack_code": {
-            "maxLength": 5,
+            "maxLength": 6,
```

Comment:
I think this should be fine, as the ThronesDB schema seems to allow 255 characters for the `code` column on packs. But not 100% certain.